### DISCUSSION TITLE
I want to deploy an observable framework dashboard within output of another static site

### DISCUSSION THREAD

**Discussion by Rahuketu86** [observablehq/framework#1094] [Q&A] [answered]
My personal static site is in quarto. I want to create a dashboard in observable and deploy it to one of the subpath on quarto site. This involves 

1. creating a docs folder for original site . 
2. Creating a build for dashboard using 'npm run build' with  following configuration . 
```
// See https://observablehq.com/framework/config for documentation.
export default {
  // The project’s title; used in the sidebar and webpage titles.
  title: "Device Dashboard",
  output: "../../_docs/apps/device_dashboard", // path to the output root for build
  // search: true, // activate search
};
```

3.  Uploading this folder to a different branch gh-pages 
4. Deploying this branch to my static host : netlify/ vercel/ render etc...

When doing this I can see index page correctly. 
<img width="1656" alt="image" src="https://github.com/observablehq/framework/assets/786803/334034bf-f587-4ab5-98fc-f039c7cb8e78">

However subpages give 404 error
<img width="626" alt="image" src="https://github.com/observablehq/framework/assets/786803/6c34bd81-5993-4377-84f4-2dfe99cc9ca3">

Please advise the right workflow to achieve this

**Comment by mbostock** [comment, marked as answer]
Did you try setting the cleanUrls option to false in your config? This could be because your static server doesn’t support clean URLs (dropping the .html extension).

**Reply by Rahuketu86** [reply]
Thank you cleanUrl settings fixes the issue
